recipes/rotor/all/conanfile.py

```diff
@@ -126,3 +132,7 @@ def package_info(self):
         if self.options.get_safe("enable_ev", False):
             self.cpp_info.components["ev"].libs = ["rotor_ev"]
             self.cpp_info.components["ev"].requires = ["core", "libev::libev"]
+
+        if self.options.get_safe("enable_fltk", False):
+            self.cpp_info.components["fltk"].libs = ["rotor_fltk"]
+            self.cpp_info.components["fltk"].requires = ["core", "fltk::fltk"]
```

Comment:
```suggestion
            self.cpp_info.components["fltk"].requires = ["core", "fltk::fltk"]
        if self.settings.os in ["Linux", "FreeBSD"]:
            self.cpp_info.components["core"].system_libs.append("m")
```

Reply:
`rotor` code does not contains any `<cmath>`, it is pulled in by some other dependencies (probably, boost).